src/components/NumberInput/NumberInput.jsx

```diff
@@ -27,7 +27,7 @@ class NumberInput extends React.Component {
   }
 
   handleBlur() {
-    this.props.onValueChange(makeField(this.props.field.value, true));
+    if (this.props.onBlur) this.props.onBlur();
```

Comment:
Do we need this check here? Can the handler just be inlined, like:

```jsx
onBlur={this.props.onBlur}
```

Reply:
Oh good call. I had it in here because I initially was keeping the `onValueChange` bit, but since I got rid of it, inlining makes more sense. :+1: